frontend/src/app/upload/page.tsx

```diff
@@ -65,14 +65,18 @@ export default function Home() {
   }, [clipId]);
 
   const checkProgress = async () => {
-    const rest = await fetch(`/api/clips/${clipId}/progress`);
+    const rest = await fetch(`/api/clips/progress?cid=${clipId}`);
```

Comment:
```suggestion
    const resp = await fetch(`/api/clips/progress?cid=${clipId}`);
```
So we're consistent with naming

Reply:
was a typo, good catch